gateway/webapp/webshop/controller/Checkout.controller.js

```diff
@@ -129,7 +124,7 @@ sap.ui.define([
 		},
 		handleWizardSubmit: function () {
 			var that = this;
-			this._oWizardReviewPage.setBusy(true);
+			// this._oWizardReviewPage.setBusy(true);
```

Comment:
Why commented ?

Reply:
the page gets stuck when the wizard is completed twice. removing this solves the issue